### PR TITLE
Add "Motion Blur" video option

### LINKED ISSUE #756: Motion Blur into Video Settings
**Is your feature request related to a problem? Please describe.**

A lot people hate motion blur and would like to have it disabled on any server. This would be usefull because not every server offers options for players to change their motion blur setting.

**Describe the solution you'd like**

There should be added a bar for the strengh of motion blur effect in video settings and a tick box which allows scripts to overwrite the motion blur value and resets motion blur value on server leave.
Bar:
0 |-----------| 100
tick box:
☐ Allow scripts to overwrite Motion Blur setting.

Related script function:
setPlayerBlurLevel / setBlurLevel
https://wiki.multitheftauto.com/wiki/SetPlayerBlurLevel

### PR DESCRIPTION
**Pull request by lopezloo**
This PR adds "Motion Blur" checkbox into video settings. Enabled by default.

Player is in control until `setBlurLevel()` or `setPlayerBlurLevel()` is called. Then script is in control until disconnect, `resetBlurLevel()` or `resetMapInfo()` being called.

`dxGetStatus().SettingBlur` can be used to retrieve current player setting.

~~PR is complete but I'm marking it as draft until #2347 gets merged because that PR also adds video option and without having it merged there would be an empty space in GUI~~.

### CODE REVIEW

**Comment by Pirulax** [comment]
> Enabled by default.

I think it would be better for it to be opt-in instead. Most people hate MM.

**Comment by lopezloo** [comment]
> > Enabled by default.
> 
> I think it would be better for it to be opt-in instead. Most people hate MM.

It is enabled in Single Player though.

**Comment by lopezloo** [comment]
I'm reopening this pull request since #2347 was merged.

**Comment by TheNormalnij** [comment]
Hi. I found a bug in settings menu. Checkbox doesn't  work for some langs.
<details>
  <summary>Screenshot</summary>
    
</details>

**Comment by lopezloo** [comment]
I have rearranged positions of these options so they shouldn't overlap anymore.

File: Client/mods/deathmatch/logic/luadefs/CLuaDrawingDefs.cpp [hunk 1]
```diff
@@ -1712,8 +1712,12 @@ int CLuaDrawingDefs::DxGetStatus(lua_State* luaVM)
         lua_pushboolean(luaVM, dxStatus.settings.bHighDetailPeds);
         lua_settable(luaVM, -3);
 
+        lua_pushstring(luaVM, "SettingBlur");
+        lua_pushboolean(luaVM, dxStatus.settings.bBlur);
+
```
Comment (TheNormalnij) [inline Client/mods/deathmatch/logic/luadefs/CLuaDrawingDefs.cpp:1718]: Missing `lua_settable(luaVM, -3);`
dxGetStatus() doesn't work

**Review by TheNormalnij — APPROVED** [review]

**Pull request merged**